### ISSUE TITLE
More on 3D Pre-Training and Neural Representation Learning

### ISSUE THREAD

**Issue by hansen7** [hansen7/OcCo#10]
### Pre-Training
- "Don't Stop Pretraining: Adapt Language Models to Domains and Tasks", ACL 2020
- "Train No Evil: Selective Masking for Task-Guided Pre-Training", EMNLP 2020
- "Co-Tuning for Transfer Learning", NeurIPS 2020
- "Learning to Adapt to Evolving Domains", NeurIPS 2020

### Neural Representation Learning

Some of Vincent [Sitzmann](https://vsitzmann.github.io)'s work:
- Implicit Neural Representations with Periodic Activation Functions, NeurIPS 2020, [website](https://vsitzmann.github.io/siren/)
- Scene Representation Networks: Continuous 3D-Structure-Aware Neural Scene Representations, NeurIPS 2019
- and his thesis on "Self-supervised Scene Representation Learning"
